### PR TITLE
baseline status PR fetch and log

### PR DESCRIPTION
**Pull request by aug2uag**
@carllerche the static path in main wasn't reverted back to original, I'll wait on @tobz to get started listing out issues

### CODE REVIEW

File: src/action/status.rs [hunk 1]
```diff
@@ -1,26 +1,231 @@
...
 use crate::github;
 use crate::package;
+use crate::{Config, Workspace};
 
 use semver::Version;
 
+use slog::*;
+
+/**
+WORKSPACE STRUCTURE
```
Comment (carllerche) [inline src/action/status.rs]: Maybe include this in some doc file to avoid cluttering the source (it is a bit noisy).

File: src/main.rs [hunk 2]
```diff
@@ -1,8 +1,8 @@
-use clap::{Arg, App, SubCommand};
+use clap::{App, Arg, SubCommand};
 
 use std::path::Path;
 
-const PATH: &str = "/Users/carllerche/Code/Tokio/tokio";
+const PATH: &str = "/usr/local/var/foss/tokio";
```
Comment (carllerche) [inline src/main.rs]: What about updating the CLI to take a `--project-path` flag to specify the path instead of hard coding one? 👍

**Review by carllerche — CHANGES_REQUESTED** [review]

**Comment by aug2uag** [comment]
@carllerche I made the updates, path's now required 
`# shipit $target_directory $subcommand`

File: src/main.rs (hunk 2 above)
Comment (aug2uag) [reply src/main.rs]: would you prefer to prompt the user to input the path by passing `--project-path`? I missed this detail, sorry

**Comment by carllerche** [comment]
Thanks!

**Review by carllerche — APPROVED** [review]

**Comment by aug2uag** [comment]
Thank you Carl, it's a pleasure. I'd appreciate any tasks you can delegate my way from this or other projects.

**Pull request merged**